packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -216,33 +217,31 @@ function _consoleBreadcrumb(handlerData: HandlerData & { args: unknown[]; level:
 /**
  * Creates breadcrumbs from XHR API calls
  */
-function _xhrBreadcrumb(handlerData: HandlerData & { xhr: SentryWrappedXMLHttpRequest }): void {
-  if (handlerData.endTimestamp) {
-    // We only capture complete, non-sentry requests
-    if (handlerData.xhr.__sentry_own_request__) {
```

Comment:
This is actually checked in the "original" breadcrumb integration in browser already, so no need to check this here again IMHO:
https://github.com/getsentry/sentry-javascript/blob/develop/packages/browser/src/integrations/breadcrumbs.ts#L222

Actually the same is also true for the endTimestamp/__sentry_xhr__ fields, but there it makes more sense to guard them defensively again, I'd say.